sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/CosmosDatabaseForTest.java

```diff
@@ -93,13 +93,13 @@ public static void cleanupStaleTestDatabases(DatabaseManager client) {
                 new SqlQuerySpec("SELECT * FROM c WHERE STARTSWITH(c.id, @PREFIX)", sqlParameterList)).collectList().block();
 
         for (CosmosDatabaseProperties db : dbs) {
-            assertThat(db.getId()).startsWith(CosmosDatabaseForTest.SHARED_DB_ID_PREFIX);
+            assertThat(ModelBridgeInternal.getResource(db).getId()).startsWith(CosmosDatabaseForTest.SHARED_DB_ID_PREFIX);
```

Comment:
once `getId()` is added back we should undo this change as getId will be required method in CosmosDatabaseProperties.

Reply:
reverted change back.